semaphore/integration_tests.py

```diff
@@ -118,7 +150,7 @@ def compare_plans(plan1, plan2):
             prod_plans = get_trip_plans("prod", fromPlace, toPlace, date, arriveBy)
             dev_plans = get_trip_plans("dev", fromPlace, toPlace, date, arriveBy)
 
-            if not compare_plans(prod_plans, dev_plans):
+            if not compare_plans(prod_plans, dev_plans, local_run=False):
```

Comment:
What is `local_run` doing? I would have assumed it would check for CI or something, or have a flag that's passed in. Otherwise, it looks like this will upload the results even if you're running on your local machine.

Reply:
`local_run` is used to save the plans to the files so I can compare them in IDE with single click instead of getting this data from stdout. There is no _uploading_ involved, it's only about saving them the local file system. It's disabled by default.